jgrygierek/BatchEntityImportBundle
# Fix MatrixRecord bug with accepting objects as data value

## Description
In the recent update to `MatrixRecord`, type hints were introduced to enhance code readability and maintainability ([PR #77](https://github.com/jgrygierek/BatchEntityImportBundle/pull/77/files#diff-dc69237383e6aa03126533bd239b7a5861af9362eef5ccdaf94a185a90785f5dR35)).

However, there's an opportunity to refine the type hinting for the `$value` parameter. Currently, it is specified as `null|string|int`, but in practice, it can take on a broader range of types, such as objects, booleans, etc. This becomes particularly relevant when utilizing `EntityType` in our import configuration, where the field type may necessitate an object value. The current solution limits this flexibility.

## Steps to Test
1. Create an import configuration with an `EntityType` field.
2. Verify that the import process will be done successfully with the specified field.

## Related Issues or Pull Requests
- https://github.com/jgrygierek/BatchEntityImportBundle/pull/77
